source/learn/rosetta_stone.md

```diff
@@ -0,0 +1,1673 @@
+a = array([1, 2, 3, 4, 5, 6, 7, 8, 9, 10])
+b = empty(10)
+for i in range(len(a)):
+    if a[i] > 5:
+        b[i] = a[i] - 3
+    elif a[i] > 2:
+        b[i] = 1
+    else:
+        b[i] = 0
+
+```
+:::::
+
+:::::{grid-item}
+```{code-block} Fortran
+:caption: Fortran
+
+integer :: a(10), b(10)
+a = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
+where (a > 5)
+    b = a - 3
+elsewhere (a > 2)
+    b = 1
+elsewhere
+    b = 0
+end where
+
+```
+:::::
+
+:::::{grid-item}
+```{code-block} Python
+:caption: Numpy Python
+
+from numpy import array, sum, ones, size
+a = array([1, 2, 3, 4, 5, 6, 7, 8, 9, 10])
+print sum(a)
+print sum(a[(a > 2) & (a < 6)])
+o = ones(size(a), dtype="int")
+print sum(o[(a > 2) & (a < 6)])
+
+```
+:::::
+
+:::::{grid-item}
+```{code-block} Fortran
+:caption: Fortran
+
+integer :: a(10)
+a = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
+print *, sum(a)
+print *, sum(a, mask=a > 2 .and. a < 6)
+print *, count(a > 2 .and. a < 6)
+
+```
+:::::
+
+:::::{grid-item}
+```{code-block} Python
+:caption: Numpy Python
+
+from numpy import array, dot
+a = array([[1, 2], [3, 4]])
+b = array([[2, 3], [4, 5]])
+print a * b
+print dot(a, b)
+
+```
+:::::
+
+:::::{grid-item}
+```{code-block} Fortran
+:caption: Fortran
+
+integer :: a(2, 2), b(2, 2)
+a = reshape([1, 2, 3, 4], [2, 2], order=[2, 1])
+b = reshape([2, 3, 4, 5], [2, 2], order=[2, 1])
+print *, a * b
+print *, matmul(a, b)
+/
+
+```
+:::::
+
+:::::{grid-item}
+```{code-block} Python
+:caption: Numpy Python
+
+[[ 2  6]
+ [12 20]]
+[[10 13]
+ [22 29]]
+
+```
+:::::
+
+:::::{grid-item}
+```{code-block} Fortran
+:caption: Fortran
+
+2          12           6          20
+10          22          13          29
+
+```
+:::::
+
+:::::{grid-item}
+```{code-block} Python
+:caption: Numpy Python
+
+from numpy import array, pi
+a = array([i for i in range(1, 7)])
+b = array([(2*i*pi+1)/2 for i in range(1, 7)])
+c = array([i for i in range(1, 7) 
+    for j in range(1, 4)])
+
+```
+:::::
+
+:::::{grid-item}
+```{code-block} Fortran
+:caption: Fortran
+
+use types, only: dp
+use constants, only: pi
+integer :: a(6), c(18)
+real(dp) :: b(6)
+integer :: i, j
+a = [ (i, i = 1, 6) ]
+b = [ ((2*i*pi+1)/2, i = 1, 6) ]
+c = [ ((i, j = 1, 3), i = 1, 6) ]
+
+```
+:::::
+::::::
+
+### Some indexing examples
+
+<table>
+<colgroup>
+<col style="width: 49%" />
+<col style="width: 50%" />
+</colgroup>
+<tbody>
+<tr class="odd">
+<td>NumPy</td>
+<td><blockquote>
+<p>Fortran</p>
+</blockquote></td>
+</tr>
+<tr class="even">
+<td><pre><code>from numpy import array
+a = array([1, 2, 3])
+b = a
+print a[:]
+print b[:]
+print a[:2]
+print b[:2]</code></pre>
+<p>Output:</p>
+<pre><code>[1 2 3]
+[1 2 3]
+[1 2]
+[1 2]</code></pre></td>
+<td><pre class="fortran"><code>integer :: a(3), b(-1:1)
+a = [1, 2, 3]
+b = a
+print *, a(:)
+print *, b(:)
+print *, a(:2)
+print *, b(:0)</code></pre>
+<p>Output:</p>
+<pre><code>1           2           3
+1           2           3
+1           2
+1           2</code></pre></td>
+</tr>
+</tbody>
+</table>
```

Comment:
Do you plan to switch these to use the markdown like you do for all examples above?

Reply:
Awesome!